packages/history-utility/src/history-utility.ts

```diff
@@ -91,7 +102,10 @@ const deepClone = <T>(value: T): T => {
  *   count: 1,
  * })
  */
-export function proxyWithHistory<V>(initialValue: V, skipSubscribe = false) {
+export function proxyWithHistory<V>(
+  initialValue: V,
+  skipSubscribeOrCallback: SkipSubscribeOrCallback = false
```

Comment:
You may consider making the second argument option object for future extension.

Reply:
yup true